src/diffusers/__init__.py

```diff
@@ -29,14 +36,12 @@
     )
     from .pipeline_utils import DiffusionPipeline
     from .pipelines import (
-        AudioDiffusionPipeline,
```

Comment:
why remove this?

Reply:
It's now under `is_torch_available() and is_librosa_available()` instead of just `is_torch_available()`

Reply:
ah it's further down